database/postgres/connection.go

```diff
@@ -16,9 +17,12 @@ func Connect(ctx context.Context, dsnURI string) (*pgxpool.Pool, error) {
 		return nil, dsn.RedactParseError(err)
 	}
 	poolCfg.AfterConnect = func(ctx context.Context, conn *pgx.Conn) error {
-		_, err := conn.Exec(ctx, "SET search_path=public")
-		if err != nil {
-			return err
+		// force a known search_path if DSN doesn't specify one
+		if !strings.Contains(dsnURI, "&search_path=") && !strings.Contains(dsnURI, "?search_path=") {
```

Comment:
maybe it's better to parse dsn as URI instead of trying a simple text search?

Reply:
Would add complexity and we only want to check if they have a `search_path` in there.